pandas/io/excel.py

```diff
@@ -168,7 +168,20 @@ def get_writer(engine_name):
         raise ValueError("No Excel writer '%s'" % engine_name)
 
 
-@Appender(_read_excel_doc)
+def split_doc(doc_string, word_wrap=80):
```

Comment:
Yeah, moving it to a common location is better, I just meant the current function (after moving) is fine.

wrap is not doing exactly the same as this does not take account of the indentation needed in this case